src/components/alert/alert.js

```diff
@@ -150,18 +152,17 @@ export const BAlert = /*#__PURE__*/ Vue.extend({
       $alert = h(
         'div',
         {
-          key: this._uid,
-          staticClass: 'alert',
+          staticClass: CLASS_NAME,
           class: {
-            'alert-dismissible': this.dismissible,
-            [`alert-${this.variant}`]: this.variant
+            [`${CLASS_NAME}-dismissible`]: dismissible,
+            [`${CLASS_NAME}-${variant}`]: !!variant
```

Comment:
Could we maybe make a utility to generate the concatenated class name?

```js
const suffixClass = (base, suffix) => {
  return [base, suffix].filter(identity).join('-')
}
```

Which would reduce the number of transpiled concats: `'base'.concat('-dismissable')`?

Reply:
or maybe:

```js
const suffixClass = (...strings) => {
  return strings.filter(identity).join('-')
}
```

Reply:
Place it inside dom utils?

Reply:
Hmmm.... or maybe a generic string utility in `utils/string`?

Reply:
maybe dom, and call it `buildClass`?

Reply:
or `classConcat`